### ISSUE TITLE
Improve CRX install instructions for Windows

### ISSUE THREAD

**Issue by qsniyg** [qsniyg/maxurl#821]
Thanks to Liz on discord for reporting.

Apparently this is needed:

```
Windows Registry Editor Version 5.00
[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Google\Chrome\ExtensionInstallWhitelist]
"1"="momhpkepmajdopjgahiglmboldkepibg"

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge\ExtensionInstallAllowlist]
"1"="momhpkepmajdopjgahiglmboldkepibg"
```

**Comment by Dhyfer1** [comment]
After loading the crx file I can't activate it in Edge, and I don't see all the path indicated in your previous comment, only what you see in the image. This is what the warning (in red color) says: _This extension does not come from any known source and may have been added without your knowledge_.

![image_1](https://user-images.githubusercontent.com/50301201/128620672-f24ef8b2-cfe3-4eef-94ab-6600a242d8fa.png)

![image_2](https://user-images.githubusercontent.com/50301201/128620681-d658272f-7817-4d0c-a36c-6d32594f4baf.png)

Is there any way to activate the extension?

**Comment by qsniyg** [comment]
I'm not entirely sure, but perhaps create the `Edge\ExtensionInstallAllowlist` folders, then set the `1` key to the string value `momhpkepmajdopjgahiglmboldkepibg`?

I'm not too familiar with windows, so I'm not sure if this would work or not.

**Comment by IZeeLikeEyeZee** [comment]
@qsniyg Yeah, the solution you posted is what a registry file to set the keys/values/data, which would automatically get created if missing when the .reg file is ran. You can include a file like that in the crx install instructions, similar to [how TTV LOL PRO does it](https://github.com/younesaassila/ttv-lol-pro/tree/v1#-chromium-chrome-edge-or-brave). Also I suggest setting the key's name to a different/higher number than 1, since most extensions' instructions use that, and it could avoid accidental overwriting for less savvy users 🙂
